eth_worker/eth_manager/__init__.py

```diff
@@ -57,7 +57,7 @@
 celery_app.conf.beat_schedule = {
     "maintain_eth_balances": {
         "task": utils.eth_endpoint('topup_wallets'),
-        "schedule": 600.0
```

Comment:
Good debug time, but should probably change this back before merging?

Reply:
hmmmm i'm not sure. Topup wallets is less critical than it used to be, because all tasks automatically check to see if they have enough balance before transacting. So then the main utility of this is that it keeps txn time low by ensuring a task doesn't need to wait for topup before running. In this regard, even just once a day would probably be sufficient... maybe..